### DISCUSSION TITLE
[CFP] 44 spotkanie Microsoft Azure User Group Poland

### DISCUSSION THREAD

**Discussion by kaluzaaa** [AzureCommunityPL/group#130] [Ogólna]
18.11 kolejny meetup naszej społeczności‼️ Chcesz wystąpić z prezentacją? Nic trudnego!

Do 05.11 do 22:00 można zgłosić swoją sesję!

[Formularz CFP](https://azurecommunitypl.typeform.com/to/JlI0UojR)

**Comment by mgrabarz** [comment]
Meetup już założony, można się dopisywać: https://www.meetup.com/Microsoft-Azure-Users-Group-Poland/events/274337933/
